packages/dx-react-scheduler/docs/guides/views.md

```diff
@@ -0,0 +1,33 @@
+# React Scheduler - Views
+
+The Scheduler component provides several types of views. Views display information in various ways, you can choose the most convenient view for your data source or combine few views and switching between them.
+
+## Related Plugins
+
+The following plugins render corresponding views:
+
+- [DayView](../reference/day-view.md) - renders a day layout
+- [WeekView](../reference/week-view.md) - renders a week layout
+- [MonthView](../reference/month-view.md) - renders a month layout
```

Comment:
I'm not sure, but what about the `ViewState` one? It affects all views. Its name starts with `View`, at least))

Reply:
This plugin is not required for basic setup. I think we should not add it.